services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -355,4 +379,5 @@
   (let [identifier (complete-permanent-id-request (:shortUsername current-user)
                                                   (admin-get-permanent-id-request request-id nil))]
     (update-permanent-id-request request-id nil (json/encode {:status       status-code-completion
+                                                              :comments     identifier
```

Comment:
This status update is an automated `Completion` status update.

The `comments` field can be any string and is the comments of the status update, which is displayed in the request's status history and also used for the body of the status update's notification. So the comments also become the body of the notification's email.
Other status updates besides the `Completion` status can set any other arbitrary string for the comments, and those mainly come from the UI when an admin is updating the status of a request.

The `permanent_id` is the newly created ID. This field is specific to the `POST /admin/permanent-id-requests/{request-id}/status` metadata endpoint and is only sent once when the request is completed, then the metadata service saves the ID with the Permanent ID Request records. This endpoint will return an error if this field is populated when an ID is already saved for the Permanent ID Request.